### PR TITLE
User feedback footer

### LINKED ISSUE #102: Implement way to receive user feedback
**Description**

Right now, if a user found a bug or is having trouble doing something, there isn't really a way for us to get notified about it.
Well I guess technically there is: student finds bug -> tells prof -> tells ramon -> tells us.
However, for every bug that gets reported this way, there are many more that are likely not going to be reported due to it taking time out of their day to tell us (e.g. how often have you gone out of your way to contact someone about a bug in their page).

Therefore, we should implement a quick way for users to submit feedback.

I'm thinking maybe adding a footer bar at the bottom of each page saying "found a bug or having issues? Contact us at [email or a textbox that when they hit 'send' will send us an email]"

If we really wanted to get fancy, we could implement something akin to how YouTube does it, where on top of a textbox, you can choose to submit a screenshot of your browser window (which then prompts the user to censor anything they'd like). Might be overkill, at least initially, but seeing as how most users don't always know how to describe a bug (e.g. "your page is broken"), this may be worthwhile.

This issue is especially important with all the new users coming on (and that it hasn't been tested on this large of a scale yet and with all the new features)

### PR DESCRIPTION
**Pull request by AdamFipke**
# Description

Just sticks a footer bar on the end of the page with my work email on it.

As a bonus this also changes the tag color of the role on /courses so that it's not all green.

Closes #102 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This requires a run of `yarn install`

# How Has This Been Tested?

Since this required changing everything to flex, I checked EVERY page on desktop on mobile to make sure nothing broke, and it looks like everything is working as expected.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code where needed 
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any work that this PR is dependent on has been merged into the main branch
- [x] Any UI changes have been checked to work on desktop, tablet, and mobile

### CODE REVIEW

**Comment by AdamFipke** [comment]
I'm just requesting a review from everyone just 'cause I guess I should get in the habit of this. You don't need to review the code, just asking in case there are general comments or questions etc. about it

**Comment by AdamFipke** [comment]
bro what the tests failed

**Comment by AdamFipke** [comment]
oh okay this is just one of those 1/20 times where some deadlock error or something breaks the tests

File: packages/frontend/app/(dashboard)/components/coursesSection.tsx [hunk 1]
```diff
@@ -61,10 +61,19 @@ const CoursesSection: React.FC<CoursesSectionProps> = ({ courses }) => {
               </div>
             }
           >
-            <div className="flex items-center justify-between align-middle">
+            <div className="flex flex-wrap items-center justify-between align-middle">
               {/* TODO: add the course section to the name since out of all the places to put the section, it should probably show up here. */}
```
Comment (d3li0n) [inline packages/frontend/app/(dashboard)/components/coursesSection.tsx:65]: Do we need this comment since it's in Meta?

Comment (AdamFipke) [reply packages/frontend/app/(dashboard)/components/coursesSection.tsx:65]: the course section is not shown no. The endpoint needs to be changed to return the course section

File: packages/frontend/app/(dashboard)/layout.tsx [hunk 2]
```diff
@@ -73,12 +74,13 @@ const Layout: React.FC<LayoutProps> = ({ children }) => {
           URLSegments[4] === 'chatbot_questions' ? (
             <div className="p-1">{children}</div>
           ) : (
-            <StandardPageContainer className="min-h-full">
+            <StandardPageContainer className="">
```
[the remark this answers is not in the document]
Comment (AdamFipke) [reply packages/frontend/app/(dashboard)/layout.tsx:77]: Please use github's code suggestion tool, that way i can just click "commit suggestion". It is not worth my time to checkout out over to there to fix this tiny thing

Comment (d3li0n) [inline packages/frontend/app/(dashboard)/layout.tsx:77]:
```suggestion
            <StandardPageContainer>
```

As per comment :)

Comment (AdamFipke) [reply packages/frontend/app/(dashboard)/layout.tsx]: hehe thanks!

File: packages/frontend/app/(dashboard)/components/coursesSection.tsx (hunk 1 above)
Comment (d3li0n) [reply packages/frontend/app/(dashboard)/components/coursesSection.tsx:65]: Ah, I misread it as course title, not section :/

**Review by d3li0n — APPROVED** [review]

**Pull request merged**